extraConfigSriov.py

```diff
@@ -240,6 +264,27 @@ def ExtraConfigSriovOvSHWOL_NewAPI(cc: ClustersConfig, _: ExtraConfigArgs, futur
     ensure_pci_realloc(cc, client, "sriov")
 
 
+def _check_sriov_installed(client: K8sClient) -> None:
+    for _ in range(10):
+        result = client.oc("get csv -n openshift-sriov-network-operator -o custom-columns=NAME:.metadata.name,PHASE:.status.phase")
+        if result.out:
```

Comment:
There will always be a result, this is a bit redundant:

i.e.
# oc get csv -n made-up-ns -o custom-columns=NAME:.metadata.name,PHASE:.status.phase
NAME   PHASE
# 

We might have a header row with no output after. Maybe it makes sense to just check if "Succeeded" in out